DatabaseMigrations/2020-04-24-04-AddGamePlayerStatTable.cs

```diff
@@ -0,0 +1,39 @@
+using FluentMigrator;
+
+namespace bballsim.simulation.DatabaseMigrations
+{
+    [Migration(2020042404)]
+    public class M_2020_04_24_04_AddGamePlayerStat : Migration
```

Comment:
I wonder if we should be using dates for these scripts. Maybe a simple incrementer of 1, 2, 3... would work better. I just wonder because I'd rather not have to rename a class or script because the date changed before I merged it. If the incrementer was just a sequence, then I do not have to rename it unless a different script gets in first.

Reply:
> I wonder if we should be using dates for these scripts. Maybe a simple incrementer of 1, 2, 3... would work better. I just wonder because I'd rather not have to rename a class or script because the date changed before I merged it. If the incrementer was just a sequence, then I do not have to rename it unless a different script gets in first.

Please comment on this before merging. It is not a required change, but I'd like to discuss it before approving and merging.